Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIScripting.mm

```diff
@@ -473,6 +473,47 @@
     [manager loadAndRun];
 }
 
+TEST(WKWebExtensionAPIScripting, World)
+{
+    TestWebKitAPI::HTTPServer server({
+        { "/"_s, { { { "Content-Type"_s, "text/html"_s } }, "<script> const world = 'MAIN'; </script>"_s } }
+    }, TestWebKitAPI::HTTPServer::Protocol::Http);
+
+    auto *backgroundScript = Util::constructScript(@[
+        @"const tabs = await browser?.tabs?.query({ active: true, currentWindow: true })",
+        @"const tabId = tabs[0].id",
+
+        @"function testWorld() {",
+        @"  if (world)",
+        @"    browser.test.notifyPass()",
+        @"  else",
+        @"    console.log(undefined)",
```

Comment:
Does this console.log have any affect? Should we be marking this as a failure instead? Or is that what causes the JavaScript exception?

Reply:
A better way to throw an exception is `throw 'Foo Message'`.

Reply:
Or `throw new Error('Message')`.